### ISSUE TITLE
Building the Cpp target with VS is noisy

### ISSUE THREAD

**Issue by DefaultRyan** [antlr/antlr4#2539]
When I build the cpp runtime target, I'm getting slammed with a pile of these warnings:
`Command line warning D9025: overriding '/W3' with '/W4'`
`Command line warning D9025: overriding '/MDd' with '/MTd'`

I expect this project to build cleanly so that I can use it in my own project without polluting my build logs with a ton of noise that's not my fault.

**Comment by Nic30** [comment]
I believe you are using wrong release type or something, this messages should be silenced by default.
(I started to use cmake, but I did not seen this messages with pure vs2017 https://ci.appveyor.com/project/nic30/hdlconvertor)

**Comment by DefaultRyan** [comment]
I should clarify. I'm building it with the VS compiler in cmake. All I did was simply clone the repo and attempt to build the cpp runtime.

**Comment by Nic30** [comment]
Are you using this CMakeLists.txt from runtime/Cpp and not from  runtime/Cpp/runtime right?

**Comment by DefaultRyan** [comment]
Yes, I just now re-cloned the repo, and from within VS2017 I opened runtime/Cpp/CMakeLists.txt and from the CMake menu selected "Build only" -> "antlr4-runtime-static.lib". This was using all the default VS settings and building the x86-Debug configuration.

The build ultimately succeeded, but all those command line warnings still come out.
